### ISSUE TITLE
Filters must be a list of filters - bug on python bindings

### ISSUE THREAD

**Issue by krystianMoras** [cozodb/cozo#119]
Hi, I wanted to try out FTS, I'm using pycozo package (version: 0.7.0)

Commands I run in order (I took them from https://github.com/cozodb/cozo/blob/b7213a733a1ed2c6930e77fec93e2393f28b4e70/cozo-core/src/runtime/tests.rs#LL907C1 )

```py
client.run(r":create a {k: String => v: String}")
client.run("""
     ?[k, v] <- [['a', 'hello world!'], ['b', 'the world is round']] 
     :put a {k => v}
""")
```
No errors, table is created successfully

However when I try to create an FTS index, no configuration of filters works:

```py
client.run("""::fts create a:fts {
            extractor: v,
            tokenizer: Simple,
            filters: [Lowercase, Stemmer('English'), Stopwords('en')]
        }""")
 ```
 ```
 QueryException: Filters must be a list of filters
 ```
 ```py
 client.run("""::fts create a:fts {
            extractor: v,
            tokenizer: Simple,
            filters: []
        }""")
 ```
  ```
 QueryException: Filters must be a list of filters
 ```
 
 Is that query not correct?

**Comment by krystianMoras** [comment]
I see that it is a duplicate now - https://github.com/cozodb/cozo/issues/90

**Comment by zh217** [comment]
It's fixed, please update to v0.7.1
